rs/registry/canister/src/registry.rs

```diff
@@ -873,7 +887,9 @@ mod tests {
         );
 
         assert_eq!(
-            registry.count_fitting_deltas(1, MAX_REGISTRY_DELTAS_SIZE - 1),
+            // Subtract 1 to prevent `mutation2` from fitting into the fitting deltas,
+            // and another 1 to account for the `timestamp_seconds` tag in HighCapacity structs.
```

Comment:
It's really unclear how this makes the difference here.  Some explanation would be helpful for future test maintainers.